lib/types.ts

```diff
@@ -41,6 +52,7 @@ export interface PluginResponse {
   plugin: PluginMetadata;
   package: any;
   manifestFiles: ManifestFile[];
+  binHashFiles: BinHashFile[];
```

Comment:
note this affects the static scanning as well

Reply:
also I think this should be some how merged with the ```hashes``` that the static scanning response has?

Reply:
not sure what is `hashes` in static, I will check

Reply:
agreed to merge both dynamic/static data structures